docs/02-requirements-in-architecture-evaluation/02-learning-goals.adoc

```diff
@@ -0,0 +1,79 @@
+=== {learning-goals}
+
+// tag::DE[]
+[[LZ-2-1]]
+==== LZ 2-1: Qualitative Anforderungen untersuchen und behandeln
+
+Bedeutung von qualitativen Anforderungen für die Architektur erklären.
```

Comment:
Vermutlich sollen das hier auch Aufzählungen sein, oder? Dann solltest du auch hier einen Asterisk vorne dran schreiben. Sonst wirft Asciidoc dir das alles in eine Zeile.